Gems/ScriptCanvas/Code/Editor/View/Windows/ScriptEventMenu.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <ScriptCanvas/Core/Core.h>
+
+namespace ScriptEvents
+{
+    namespace Editor
```

Comment:
FYI: The C++17 nested namespace declaration can be used here to save an indentation level
```suggestion
namespace ScriptEvents::Editor
{
```